scripts/python_compliance.sh

```diff
@@ -46,6 +46,20 @@ else
     exit 1
 fi
 
+if $MAMBA_CMD run -n modyn ruff check experiments benchmark/arxiv_kaggle benchmark/huffpost_kaggle ; then
+    echo "No ruff check errors"
+else
+    echo "ruf check errors"
+    exit 1
+fi
+
+if $MAMBA_CMD run -n modyn ruff format experiments benchmark/arxiv_kaggle benchmark/huffpost_kaggle ; then
```

Comment:
do we need to supply the ruff.toml file or does it get that automatically?

Reply:
It should be able to pick it up automatically. Unless the script is executed from somewhere outside of the repo. Vut I guess we would notice if that happens.